extensions/wikia/ImageReview/ImageListGetter.class.php

```diff
@@ -315,4 +318,54 @@ private function getInReviewState( int $state ) : int {
 
 		else return ImageReviewStates::IN_REVIEW;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)